modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/RunDbNSFP.pm

```diff
@@ -0,0 +1,258 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=cut
+package Bio::EnsEMBL::Variation::Pipeline::ProteinFunction::RunDbNSFP;
+
+use strict;
+use Bio::DB::HTS::Tabix;
+use Bio::Tools::CodonTable;
+use File::Path qw(make_path remove_tree);
+use Data::Dumper;
+use FileHandle;
+use Bio::EnsEMBL::Registry;
+use Bio::EnsEMBL::Variation::ProteinFunctionPredictionMatrix qw(@ALL_AAS);
+
+use base ('Bio::EnsEMBL::Variation::Pipeline::ProteinFunction::BaseProteinFunction');
+
+my $CADD_CUTOFF = 0.0;
+my $REVEL_CUTOFF = 0.5;
+sub run {
```

Comment:
I know further polishing is coming - lots of comments would be useful in here at that point.

Reply:
I will add documentation after branch.